### ISSUE TITLE
vertaisarvionti

### ISSUE THREAD

**Issue by eevahanka** [tietotuomas/Kirjoitusvirheiden-korjaaja#3]
27.4. klo 7:30

Todella mielenkiintoinen projekti vaikuttaa olevan todella hyvällä mallilla. Koodia on helppo lukea.

damerau_levenshtein.py -tiedoston #-kommentit olisi hyvä muuttaa docstringiksi, jota muista tiedostoista jo löytyy, tällä hetkellä tiedosto on vaikeasti luettavissa kommenttien takia. Ko tiedoston lopussa on vanhaa koodia (?) kommentoituna pois käytöstä, oletan että se ei ole jäämässä sinne lopullisesti. Tätä tiedostoa on taidettu muokata viimeksi eilen illalla niin se myös saattaa vain olla kesken.

Ehkä hieman pilkun hinkkausta mutta itse pidän koodin staattisen analyysin github actioneissa suoritettavana. Ja samalla linjalla vielä: koodisi ei tällä hetkellä taida läpäistä staattista analyysiä, joka on toki pieni asia.

Testit vaikuttavat kattavilta ja testitapaukset hyviltä.

**Comment by tietotuomas** [comment]
Joo, tarkoitukseni siistiä vielä  damerau_levenshtein.py -tiedosto. Pylintin suoritus hajotti jostain syystä CI:n, pitää yrittää sitäkin uudemman kerran ennen lopullista palautusta. 

Kiitos kommenteista ja hyvää vappua!
